libs/react/src/lib/list/listItem.tsx

```diff
@@ -0,0 +1,17 @@
+import { ListType } from '@sebgroup/extract'
+import React, { HTMLAttributes, ReactNode } from 'react'
+
+interface ListItemProps extends HTMLAttributes<HTMLLIElement> {
+  listType?: ListType
+  children?: ReactNode
+}
+
+const ListItem = ({ listType, children, ...props }: ListItemProps) => {
+  if (listType === 'table') {
+    return <div>{children}</div>
+  } else {
+    return <li {...props}>{children}</li>
+  }
```

Comment:
I guess you will have to add another condition to wrap the children in `<dl>` as well

Reply:
Yeap, will have to look into how to implement that. The `<div>` on line 11 is used to style the table that's why its there

Reply:
I made the changes and I saw that the tag was changed from `<dl>` to `<dd>` in another PR so I made the changes accordingly